monitoring/querying-metrics.adoc

```diff
@@ -0,0 +1,39 @@
+:_content-type: ASSEMBLY
+[id="querying-metrics"]
+= Querying metrics
+include::_attributes/common-attributes.adoc[]
+:context: querying-metrics
+
+toc::[]
+
+You can query metrics to view data about how cluster components and your own workloads are performing.
+
+// Querying metrics
+include::modules/monitoring-about-querying-metrics.adoc[leveloffset=+1]
+//include::modules/monitoring-contents-of-the-metrics-ui.adoc[leveloffset=+2]
+include::modules/monitoring-querying-metrics-for-all-projects-as-an-administrator.adoc[leveloffset=+2]
+
+[role="_additional-resources"]
+.Additional resources
+
+* For more information about creating PromQL queries, see the link:https://prometheus.io/docs/prometheus/latest/querying/basics/[Prometheus query documentation].
+
+include::modules/monitoring-querying-metrics-for-user-defined-projects-as-a-developer.adoc[leveloffset=+2]
+
+[role="_additional-resources"]
+.Additional resources
+
+* For more information about creating PromQL queries, see the link:https://prometheus.io/docs/prometheus/latest/querying/basics/[Prometheus query documentation]. 
+
+include::modules/monitoring-exploring-the-visualized-metrics.adoc[leveloffset=+2]
+
+[role="_additional-resources"]
+.Additional resources
+
+* See xref:../monitoring/querying-metrics.adoc#about-querying-metrics_querying-metrics[Querying metrics] for details on using the PromQL interface
+* See xref:../monitoring/querying-metrics.adoc#querying-metrics-for-all-projects-as-an-administrator_querying-metrics[Querying metrics for all projects as an administrator] for details on accessing metrics for all projects as an administrator.
+* See xref:../monitoring/querying-metrics.adoc#querying-metrics-for-user-defined-projects-as-a-developer_querying-metrics[Querying metrics for user-defined projects as a developer] for details on accessing non-cluster metrics as a developer or a privileged user.
+
+== Next steps
```

Comment:
`== Next steps` requires a unique ID, as described in https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#anchoring-prerequisites-additional-resources-and-next-steps-titles-in-assemblies